CHANGES.rst

```diff
@@ -37,6 +37,9 @@ Bug Fixes
 Cubeviz
 ^^^^^^^
 
+- BUGFIX: Replaced deprecated FILETYPE header keyword with EXP_TYPE to identify JWST cubes to
+  apply an MJDOBS header patch [#1471]
```

Comment:
You don't need "bugfix" as it is already under the bug fix section. Also, user will not know what "apply patch" really mean, so I suggested alternate verbiage. Thanks!

```suggestion
- Replaced deprecated FILETYPE header keyword with EXP_TYPE to identify JWST cubes
  for proper MJD_OBS handling. [#1471]
```